extensions/amp-story/1.0/page-advancement.js

```diff
@@ -482,11 +528,9 @@ class ManualAdvancement extends AdvancementConfig {
    */
   maybePerformNavigation_(event) {
     const target = dev().assertElement(event.target);
+    const pageRect = this.element_./*OK*/getBoundingClientRect();
```

Comment:
`this.getLayoutBox()` should give you the equivalent cached positions from the runtime.